pages/micromaster/profiles/Farzankh.md

```diff
@@ -0,0 +1,26 @@
+# Farzan Khoobchehr
+
+* **Location:** San Diego, CA | PDT
+
+* **OS:** Windows 10
+
+Hello everyone!
+My name is Farzan and I am a software engineer student at University of California, Irvine. I like coding, hiking, riding bicycle, playing basketball and walking by the beach.
+
+## Relevant Skills:
+
+| Programming Languages |
+|---|
+| Java |
+| Python |
+| JS |
+| React.js |
+
+Besides, I have experience in HTML, CSS, AWS, MySQL, MongoDB, Android Development.
+
+To leanrn more about me you can check my website at https://farzan.codes
+***
+
+
+
+PS: I love *Pizza, Ice Cream and Chocolate cake!* :pizza: :ice_cream: :chocolate_bar: :cake:
```

Comment:
Instead of using tags please copy paste actual code, as these icon codes does not always work.

Reply:
I made the changes